ts/components/SessionQRCode.tsx

```diff
@@ -33,6 +33,7 @@ export type SessionQRCodeProps = {
   ariaLabel?: string;
   dataTestId?: SessionDataTestId;
   style?: CSSProperties;
+  children?: React.ReactNode;
```

Comment:
Not a big deal but we could extend the react import to import ReactNode so we avoid global imports, not sure how strict we've been on this for desktop but that's what I tend to try and do with the website stuff.